tools/integration_tests/util/mounting/dynamic_mounting/dynamic_mounting.go

```diff
@@ -40,7 +40,6 @@ func MountGcsfuseWithDynamicMounting(flags []string) (err error) {
 		"--debug_fs",
 		"--debug_fuse",
 		"--log-file=" + setup.LogFile(),
-		"--log-format=text",
```

Comment:
Assuming the default value log-format = json. 

Should we pass the config containing log-format=text? Just want to make sure, we are not changing any behavior. Also, `text` format is easy to analyze (manually).

Reply:
Discussed offline that it's okay to not pass any default log-format flag as it creates confusion in cases where config file is also passed with log format but flag takes precedence.